pycryptobot.py

```diff
@@ -1392,6 +1392,11 @@ def executeJob(
 
             _state.last_df_index = str(df_last.index.format()[0])
 
+            if _app.enabledLogBuySellInJson() == True \
+                    and _state.action in ["BUY", "SELL"] \
+                    and len(_app.trade_tracker) > 0:
+                Logger.info( _app.trade_tracker.loc[len(_app.trade_tracker)-1].to_json())
```

Comment:
Would you be able to post here an example of the log?

Reply:
Also, if this config is enabled, we'll be logging 2 messages for each BUY/SELL, right? Do you think wrapping it in an if and logging only JSON or non-JSON message for buy/sell would be nice?

Reply:
```json
{
  "Datetime": "2021-10-29 15:15:00",
  "Market": "LRCUSDT",
  "Action": "BUY",
  "Price": 0.5176,
  "Base": 1931.9938176198,
  "Quote": 1000,
  "Margin": null,
  "Profit": null,
  "Fee": null,
  "DF_High": 0.5807,
  "DF_Low": 0.3715
}
```



<img width="1597" alt="image" src="https://user-images.githubusercontent.com/2392977/140311487-26e1ea9d-3460-43b3-8abc-57f031d06ca6.png">

Reply:
Ideally is to have two types of loggers - plain text and json, but it requires rewriting a lot of code
> logging only JSON or non-JSON message for buy/sell would be nice
it's mostly for automatic systems so I think it will be ok, plus to output only one message requires more changes

Reply:
I think ideally we would have multiple loggers and make them configurable. I tried and pictured how this could work here, but I didn't have the time to implement: 

https://github.com/whittlem/pycryptobot/issues/323#issuecomment-897789589

This would clean up the code a lot if we just had to say `log.error('foobar')` instead of `if telegram.enabled: do this; if json.enabled: do that...`

Reply:
It could be also events system and configurable loggers that read this events
Also it will be useful for telegram bot

Reply:
@loomsen what do you think about [pymitter](https://pypi.org/project/pymitter/)
looks like it's exactly what needed
i can try to use it for logging